features/cellular/framework/API/CellularNetwork.h

```diff
@@ -1,11 +1,4 @@
 /*
- * Copyright (c) 2017, Arm Limited and affiliates.
```

Comment:
The copyright header is no longer needed?

Reply:
It should be

Reply:
Fixed.